weave/trace_server/calls_query_builder.py

```diff
@@ -65,6 +65,9 @@ def as_sql(
     def as_select_sql(self, pb: ParamBuilder, table_alias: str) -> str:
         return f"{self.as_sql(pb, table_alias)} AS {self.field}"
 
+    def is_heavy(self) -> bool:
```

Comment:
seems like this is a property of `CallsMergedField`. not sure why it needed to be hoisted up to QueryBuilderField

Reply:
you resolved this conversation without any comment or code change?